### PR TITLE
fix: skip autogen with a00 ied

### LINKED ISSUE #247: Autogen-substation: A00 can be added to the substation section
As an engineer I would like to see IED’s starting with SUBSTAITONNAME__AA00 are not located in the substation/voltagelevel/bay. 
So I can clearly see that the IED is used in a central location.

Old:

image

New:

image

### PR DESCRIPTION
**Pull request by Stef3st**
closes #247

### CODE REVIEW

File: src/compas-editors/autogen-substation.ts [hunk 1]
```diff
@@ -342,12 +346,9 @@ export default class CompasAutogenerateSubstation extends LitElement {
...
   extractNames(content: string[]) {
-    //Check for empty elements
-    const elementNames = content.filter(Boolean);
-
-    //return list of names after filtering out the duplicate elements
-    return elementNames.filter(
-      (value, index) => value && elementNames.indexOf(<string>value) === index
+    //return list of names after filtering out the empty and duplicate elements
+    return content.filter(
+      (value, index) => value && content.indexOf(<string>value) === index
```
Comment (pascalwilbrink) [inline src/compas-editors/autogen-substation.ts:351]: value is always a string, no need to cast it to a string.

**Comment by pascalwilbrink** [comment]
Is A00 case sensitive? If so, you might need to change it to this:
`!value.split(this.substationSeperator)[1].toUppercase().startsWith('A00')`
and
`filter(value => !value.toUppercase().startsWith('A00'))`

**Comment by Stef3st** [comment]
> Is A00 case sensitive? If so, you might need to change it to this: `!value.split(this.substationSeperator)[1].toUppercase().startsWith('A00')` and `filter(value => !value.toUppercase().startsWith('A00'))`

It's always uppercase. but it wouldn't hurt to add this to the code

File: src/compas-editors/autogen-substation.ts (hunk 1 above)
Comment (Stef3st) [reply src/compas-editors/autogen-substation.ts:351]: Thanks, Ill change it

**Pull request merged**